views/projects.tpl

```diff
@@ -184,24 +184,28 @@ product_tags = {
     "technical": "Links to technical information"
 }
 
-categories = {
+unsorted_categories = {
     "Privacy": [0, "Privacy Protection & Cryptography"],
     "Blockchain": [1,"Blockchains & Smart Contracts"],
     "Verification": [2, "Software Verification"],
     "Security": [3, "Device and System Security"],
     "Learning": [4, "Machine Learning"],
-    "Other": [5, "Other"],
 }
 
-applications = {
+categories = dict(sorted(unsorted_categories.items(), key=lambda k: k[1][1]))
+categories.update({ "Other": [len(categories), "Other"] })
+
+unsorted_applications = {
     "Finance": "Finance",
     "Health": "Health",
     "Gov": "Government & Humanitarian",
     "Infra": "Critical Infrastructure",
     "Info": "Digital Information",
-    "Other": "Other",
 }
 
+applications = dict(sorted(unsorted_applications.items(), key=lambda k: k[1]))
```

Comment:
you can use [OrderedDict](https://docs.python.org/3/library/collections.html#collections.OrderedDict) to avoid double  dict creation (and sorting) #nitpicking

Reply:
Thanks, it seems that OrderedDict only preserves insertion order though, I don't see how it can avoid dict creation, I still need to sort the items myself and create from it. Or maybe you have an example of what you mean?

Reply:
I've removed the intermediate variable but I think the complexity is still the same